src/controls/ScottPlot.WinForms/FormsPlot.cs

```diff
@@ -213,6 +213,7 @@ public void RefreshRequest(RenderType renderType = RenderType.LowQualityThenHigh
         private void RightClickMenu_Help_Click(object sender, EventArgs e) => new FormHelp().Show();
         private void RightClickMenu_AutoAxis_Click(object sender, EventArgs e) { Plot.AxisAuto(); Refresh(); }
         private void RightClickMenu_OpenInNewWindow_Click(object sender, EventArgs e) => new FormsPlotViewer(Plot).Show();
+        private void RightClickMenu_DetachLegend_Click(object? sender, EventArgs e) => new FormsPlotLegendViewer(this);
```

Comment:
I think you can change `object?` to `object` and the nullable warnings will go away 🤞